$OpenBSD: patch-Modules_GenerateExportHeader_cmake,v 1.1 2012/01/12 13:14:33 dcoppa Exp $

Fix wrong use of IS_ABSOLUTE
(upstream git commit 9f92cdf24d661fa46c19387fdc400c8d19c9a15d)

--- Modules/GenerateExportHeader.cmake.orig	Thu Jan 12 13:38:25 2012
+++ Modules/GenerateExportHeader.cmake	Thu Jan 12 13:39:18 2012
@@ -298,7 +298,7 @@ macro(_DO_GENERATE_EXPORT_HEADER TARGET_LIBRARY)
     set(EXPORT_MACRO_NAME ${_GEH_PREFIX_NAME}${_GEH_EXPORT_MACRO_NAME})
   endif()
   if(_GEH_EXPORT_FILE_NAME)
-    if(IS_ABSOLUTE _GEH_EXPORT_FILE_NAME)
+    if(IS_ABSOLUTE ${_GEH_EXPORT_FILE_NAME})
       set(EXPORT_FILE_NAME ${_GEH_EXPORT_FILE_NAME})
     else()
       set(EXPORT_FILE_NAME "${CMAKE_CURRENT_BINARY_DIR}/${_GEH_EXPORT_FILE_NAME}")
